src/content/reference/react/useImperativeHandle.md

```diff
@@ -166,9 +166,9 @@ input {
 
 ---
 
-### Exposing your own imperative methods {/*exposing-your-own-imperative-methods*/}
+### Mengekspos *methods* imperatif Anda sendiri {/*exposing-your-own-imperative-methods*/}
```

Comment:
```suggestion
### Mengekspos methods imperatif Anda sendiri {/*exposing-your-own-imperative-methods*/}
```

Untuk kata serapan di dalam heading, jangan di-*italic*-kan. ([Referensi](https://github.com/reactjs/id.react.dev/wiki/Panduan-Penulisan-Universal#kata-serapan-di-dalam-heading))

Reply:
Thank you for reminding @resir014 🙇, updated 🙏